### PR TITLE
update repo README

### PR DESCRIPTION
**Pull request by lvscar**
### add following information
* android runtime
* weex article 
* weex.help
* gitter.im

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -33,6 +33,11 @@ Please ***INSTALL [Git for Windows](https://git-scm.com/download/win)*** and run
     0. Click <img src="http://gtms04.alicdn.com/tps/i4/TB1wCcqMpXXXXakXpXX3G7tGXXX-34-44.png" height="16" > (`Run` button)
 0. [Add an example](./examples/README.md#add-an-example)
 
+
+#### Runtime
+
+On Android Platform , Weex code executed by [weex_v8core](https://github.com/alibaba/weex_v8core) which based on google v8 JavaScript engine.
```
Comment (luics) [inline README.md]: On Android platform, Weex code is executed in [weex_v8core](https://github.com/alibaba/weex_v8core) which is based on Google V8 JavaScript engine.

File: README.md [hunk 2]
```diff
@@ -58,6 +63,15 @@ See [SCRIPTS.md](./SCRIPTS.md) for more information.
 
 See [FAQ](http://alibaba.github.io/weex/doc/faq.html) for more information.
 
+
+### Community based weex knowledge site 
+
+* [weex article](https://github.com/weexteam/article/wiki) : article collection about Weex
```
Comment (luics) [inline README.md]: use `Weex`

File: README.md [hunk 3]
```diff
@@ -58,6 +63,15 @@ See [SCRIPTS.md](./SCRIPTS.md) for more information.
...
+* [weex.help](http://weex.help/)  :  an independent forum about Weex
```
Comment (luics) [inline README.md]: * [weex.help](http://weex.help/)  :  3rd Chinese forum

**Comment by lvscar** [comment]
thx, update again.

**Pull request merged**